gradle/libs.versions.toml

```diff
@@ -9,7 +9,7 @@ aopalliance="1.0"
 zipkinAws="1.2.5"
 logback = "1.5.16"
 wavefrontSdk="3.4.3"
-otelInstrumentationBomAlpha="2.11.0-alpha" # Brings in io.opentelemetry:opentelemetry-bom:1.44.1
+otelInstrumentationBomAlpha="2.12.0-alpha" # Brings in io.opentelemetry:opentelemetry-bom:1.44.1
```

Comment:
This is unfortunately an instance that requires manual updating. The comment should be updated to say `1.45.0` per the release notes:

> This release targets the OpenTelemetry SDK 1.45.0.

Reply:
We used to depend on different artifacts with different versioning and we ended-up having conflicts:
- https://github.com/micrometer-metrics/tracing/issues/819
- https://github.com/micrometer-metrics/tracing/pull/823

So instead of depending on multiple versions, we moved to a single source of truth (BOM). Unfortunately this BOM has another issue: its patch version upgrades contain minor version upgrades, see:https://github.com/micrometer-metrics/tracing/issues/819
We need to manually check this, hence the comment.

Another alternative I guess is removing the comment and not upgrading OTel in patch versions.